kafka_helpers/getters.go

```diff
@@ -0,0 +1,36 @@
+package kafka_helpers
+
+import (
+	"github.com/segmentio/kafka-go"
+	"internship_project/elasticsearch_helpers"
+)
+
+func NewConsumer(topicName string, EsClient elasticsearch_helpers.ElasticsearchClient) KafkaConsumer {
+	r := kafka.NewReader(kafka.ReaderConfig{
+		Brokers:   []string{"localhost:9092"},
```

Comment:
This should be a configurable parameter, similar to how database configuration is handled. Same goes for GroupID.